app/models/vote.rb

```diff
@@ -0,0 +1,12 @@
+class Vote < ApplicationRecord
+  validates :value, numericality: { only_integer: true,
+                                    odd: true,
+                                    greater_than_or_equal_to: -1,
+                                    less_than_or_equal_to: 1
+                                  }
+  validates :user, presence: true
+  validates :review, presence: true
```

Comment:
This is default in Rails 5, so you can remove the validation here

Reply:
Okay, will delete.